build/esp32/components/openweave/Kconfig

```diff
@@ -720,4 +720,35 @@ menu "OpenWeave Device Layer"
     
     endmenu
 
+    menu "Network Telemetry Options"
+    
+        config ENABLE_WIFI_TELEMETRY
+            bool "Enable WiFi Telemetry"
+            default y
+            help
+                Enable automatically uploading Wi-Fi telemetry via trait on an interval.
+        
+        config ENABLE_THREAD_TELEMETRY
+            bool "Enable Thread Telemetry"
+            default n
+            help
+                Enable automatically uploading minimal Thread telemetry and topology via trait on an interval.
+
+        config ENABLE_THREAD_TELEMETRY_FULL
```

Comment:
I think these two options might be better as a single enumerated "choice" option with three choices: NONE, MINIMAL and FULL.  See the "Device Type" option in the Kconfig.projbuild file in openweave-esp32-demo.

Reply:
Will be addressed in the follow up commits.